.circleci/config.yml

```diff
@@ -10,6 +10,24 @@ jobs:
       - checkout
       - run: tox -e flake8,unit
 
+  integration-spark-session:
+    environment:
+      DBT_INVOCATION_ENV: circle
+    docker:
+      - image: godatadriven/pyspark:3.1
```

Comment:
I was not able to reuse the fishytown test image `fishtownanalytics/test-container:10`. That image has a user `dbt_test_user` set which blocks me from `apt-get` installing because that requires root privileges.

Why is that user set in the test image and not the root user (which is Docker default)? Also, where can I find the docker file of this image? There is minimal info available on the docker hub.

Reply:
It's been a long time since we created that image, and it's not well documented. It does include (I believe) the dependencies you have listed below. See image layer 7 in [the digest on DockerHub](https://hub.docker.com/layers/fishtownanalytics/test-container/10/images/sha256-ea033e1b16eb2bc89d89d2eb79826ae9be46666975e4c77ba6fd64e95c49dfd2?context=explore)